students/shibin_mathew/lesson4/trigram.py

```diff
@@ -0,0 +1,34 @@
+
+def read_file(filename):
+    # with open(path+'/'+filename, 'r+') as f:
+    #     data = f.read()
+    list_story = []
+    f = open(filename, 'r')
+    for line in f:
+            list_story.append(line.split())
+    return list_story
+
+
+def word_library(list_story, dict_word):
+    #  Adds new words to current list from the story
+    list_punctutation = [',', '.', ';', "'", '"']
+    print(list_story)
+    for i in range(0, len(list_story)):
+        if i + 1 <= len(list_story) and i + 2 <= len(list_story):
```

Comment:
since you're getting length 3 times, might be good to extract into a var

Reply:
logically you only need to check `i + 2 <= len(list_story)` since if +2 will be less than size then + 1 will also be less than size